cmd/email-exporter/main.go

```diff
@@ -0,0 +1,107 @@
+package notmain
+
+import (
+	"context"
+	"flag"
+	"os"
+
+	"github.com/letsencrypt/boulder/cmd"
+	"github.com/letsencrypt/boulder/email"
+	emailpb "github.com/letsencrypt/boulder/email/proto"
+	bgrpc "github.com/letsencrypt/boulder/grpc"
+)
+
+// Config holds the configuration for the email-exporter service.
+type Config struct {
+	EmailExporter struct {
+		cmd.ServiceConfig
+
+		// PerDayLimit is our daily limit as determined by the tier of our
+		// Salesforce account. For more information, see:
+		// https://developer.salesforce.com/docs/marketing/pardot/guide/overview.html?q=rate%20limits
+		PerDayLimit float64 `validate:"required,min=1"`
+
+		// PardotBusinessUnit is the Pardot business unit to use.
+		PardotBusinessUnit string `validate:"required"`
+
+		// ClientId is the OAuth API client ID provided by Salesforce.
+		ClientId cmd.PasswordConfig
+
+		// ClientSecret is the OAuth API client secret provided by Salesforce.
+		ClientSecret cmd.PasswordConfig
+
+		// SalesforceBaseURL is the base URL for the Salesforce API. (e.g.,
+		// "https://login.salesforce.com")
+		SalesforceBaseURL string `validate:"required"`
+
+		// PardotBaseURL is the base URL for the Pardot API. (e.g.,
+		// "https://pi.pardot.com")
```

Comment:
Probably meant `https://api.pardot.com` here. Unless the Pardot folks happen to be really obsessed with circles.

Reply:
`pi.pardot.com` is in fact correct!

Reply:
Amaze!